src/commands/codepush/lib/react-native-utils.ts

```diff
@@ -468,11 +468,46 @@ function parseBuildGradleFile(gradleFile: string) {
     throw new Error(`Unable to find gradle file "${buildGradlePath}".`);
   }
 
+  return buildGradlePath;
+}
+
+function parseBuildGradleFile(gradleFile: string) {
+  const buildGradlePath: string = getBuildGradleFilePath(gradleFile);
   return g2js.parseFile(buildGradlePath).catch(() => {
     throw new Error(`Unable to parse the "${buildGradlePath}" file. Please ensure it is a well-formed Gradle file.`);
   });
 }
 
+// parse gradle.properties file
```

Comment:
Let's remove this comment and the two below. I don't think we need a comment for the clear code, which is easily understandable when you read it.